### ISSUE TITLE
Finding an angle for the therapeutics papers

### ISSUE THREAD

**Issue by rando2** [greenelab/covid19-review#920]
The reviewer feedback for therapeutics was totally fair -- it's kind of a laundry list of drugs right now! 

After talking with @cgreene, it seems like it will make the most sense to have appendices for each drug that are tied together with kind of a position paper that looks at how all the drug development projects intersect. That way we keep the whole history of how each drug developed intact, while still providing a reader with a crisp narrative they can follow (and read in a reasonable amount of time!)

I don't have much of a pharma or medicine background, so I would love any and all feedback on this, but looking through the whole manuscript, it appears to me that there are 4 different approaches we are describing:
1. **Clinical responses to symptoms/outcomes of COVID-19.** Giving dexamethasone for ARDS is a pretty well-known strategy, and ARDS is one of the main outcomes that is concerning in COVID-19. As @cgreene said offline, this isn't even repurposing, it's just using a drug like it was intended (as shown by the FDA not requiring an EUA). Result: Dexa is the strongest drug we have right now, and I also learned from the long-COVID project that steroids were used to treat SARS? So there may already be a significant precedent for using dexa here.
2. **Biologics that imitate what we know about what's going on.** Inflammation is a big problem with COVID-19, as illustrated by the fact that ARDS and sepsis are some of the top killers. How does the body calm down inflammation and can we do that exogenously? Similarly, people can not-get-sick when they have antibodies to something, so can we give them antibodies exogenously? These show varying amounts of promise, potentially because biologicals (other than monoclonal antibodies) seem to be a little tricky to get totally on-target?
3. **Computational approaches.** In a significant departure from previous pandemics, this time we have experience with big data & significant computational power. We have the ability to model things pretty effectively, so can we can try to brute force this problem and find things that might work by just screening lots of possibilities. Results: The jury is still out on Mpro, but I learned about a few promising screening approaches in a talk this week that I'm hoping to look into more. 
4. **Speciality treatments designed for previous killers that might share some mechanism here.** There are a few drugs where they were designed for some other awful virus (HIV or ebola, for example), that have demonstrated varying success against SARS-CoV-2. This one seems particularly reactive to me, where we're just scrambling to see if we can use groundwork already laid for previous threats that didn't totally pan out at the time. It's not really clear that any of these work super well -- there's some evidence in favor of remdesivir and some against it, and it's actually become pretty clear that using lopinavir/ritonavir for SARS probably didn't do much (and doesn't do much here).

This is super rough brainstorming from an evolutionary biologist, but I'm curious what you think! I think I will tag a small group of people to hopefully organize a stronger framework before looping in a larger number of people from the project.

@ajlee21 @RLordan @cgreene @agitter @nilswellhausen @SiminaB

**Comment by agitter** [comment]
This proposed re-organization would be a much more compelling narrative and readable manuscript.  How much do you think we could repurpose and restructure existing text versus having to write entirely new text?  Given the amount of work left on the  other manuscripts and shrinking pool of active contributors, I'm somewhat hesitant about writing a completely new review.

For computational approaches specifically, we've intentionally ignored most of the work in this area.  Even giving a high-level overview of the different types of strategies would require a lot of new reading and writing.

**Comment by rando2** [comment]
Thank you @agitter! So I think the plan is for me to try to put together a rewrite ASAP, and then get feedback from the existing list of collaborators to tighten it up for submission, rather than trying to break up the writing. In my head, I was imagining using the existing examples to illustrate points without having to go into the full story, and then we could reference the full story in the appendices. It's going to be a lot of work for me regardless, so I'd rather just start from an angle that we agree could be interesting!

I agree about my lack of reading in the computational prediction space being a possible issue here-- however, we have the nice MPro story written by @marouenbg, and @mprobson has said he can use any Wednesday 11-1 to fill in any background we need about how high-performance computing has evolved to facilitate scientific problem-solving. Casey and I also met with James Fraser this week after he gave a really elegant talk about this side of things -- and he seemed willing to reading over the manuscript and give us feedback. So I was cautiously optimistic that we could pull it off!

**Comment by ajlee21** [comment]
Thanks for taking the time to pull out these themes @rando2 !!! 

I concur with @agitter , think these sections would make a great story and make things much more clear to the reader 
(i.e. Clinical responses to symptoms/outcomes of COVID-19: "An example of this is this drug. [two sentence summary] (Appendix C))

**Comment by cgreene** [comment]
I think that the perfect is often the enemy of the good. I like the idea of repurposing text to the extent possible, and also of referring where possible to the appendix. I'm not sure that the computational approaches part really fits here since that's more a mode of finding therapeutics than a therapeutic itself. I'd lean towards 1, 2, 4 using as much text as possible from treatments that are promising.

You might also consider a fourth section for leads that did not appear to work out, which would provide an opportunity to summarize treatments that didn't go anywhere (HCQ for example). These could be more or less entirely offloaded to the appendix.

**Comment by RLordan** [comment]
I agree with @agitter, repurposing the text as much as we can. @cgreene idea to use a section for leads that did not pan out is also a good idea. They are obviously important to how we got to where we are now. As for the computational aspect, maybe a future directions/ongoing research/ discussion section could cover those aspects, if word count permits.

**Comment by rando2** [comment]
Thanks everyone for the feedback! I am attempting to do something similar for vaccines in #923. I changed up the background significantly (which took some reading) but pulled all of the illustrations directly from the existing text, which would be the plan here as well. So far I think I've put in about 1.5 work days to go from brainstorm to rough draft, so I'm hoping to clean some things up tomorrow and open that PR for comments tomorrow. 

This one would probably take a little more energy because right now therapeutics is almost 20K words long, whereas the vaccines manuscript was 14,608 words long (albeit with outstanding PRs). But my rough draft is about 6000 words, which makes me think #923 might be somewhat on track to approximate the amount of depth that a human reviewer will be able to tolerate reading 😆

**Comment by agitter** [comment]
For computational approaches, we could put together 2-3 sentences for the discussion about the breadth of approaches attempted without reviewing them:
- omic-guided repurposing via gene expression, PPI networks, etc.
- structure-based virtual screening, e.g. docking, molecular dynamics, and approximations thereof
- ligand-based virtual screening with machine learning and classic methods
- literature and knowledge graph mining

It would be nice to conclude whether computational approaches have contributed any real therapeutics ~1 year into the pandemic.  [BenevolentAI](https://www.benevolent.com/covid-19) outlines a timeline in which their knowledge graph mining identified baricitinib as a potential treatment that eventually lead to an EUA.  We could validate that story and see if there are any others.

**Comment by rando2** [comment]
I'm opening #934 and would love feedback -- it's still a draft, but I am curious what you all think about the general organization and layout. The length is in the right ballpark, as it's about 8K words right now and there's a bit more to fill in (we're aiming for 8-12K). I left the computational section essentially blank other than copying over @marouenbg's text about MPro, but I think @agitter's ideas above sound great.

I think we can get this into a submittable form with minimal additional work, if the structure seems alright to people.
